### PR TITLE
feat(editor): setup dark-mode for the code-editor, with automatic switching

### PR DESCRIPTION
**Pull request by netroy**
Bright screens at night hurt my eyes. So, I'd like to start using the dark-mode.

By default this feature is disabled. The feature can be toggled with `document.body.classList.toggle('theme-dark-beta')`

### CODE REVIEW

File: packages/design-system/src/css/_tokens.dark.scss [hunk 1]
```diff
@@ -313,6 +313,8 @@
 	}
 }
 
-body.theme-dark {
-  @include theme;
+@media (prefers-color-scheme: dark) {
+	body.dark-mode-beta {
```
Comment (mutdmour) [inline packages/design-system/src/css/_tokens.dark.scss:317]: I like that you added `beta`? but can you keep the same pattern at least.. `theme-dark-beta`.. to allow us to introduce new themes easily

File: packages/editor-ui/src/components/CodeNodeEditor/theme.ts [hunk 2]
```diff
@@ -31,80 +19,159 @@ const BASE_STYLING = {
...
+			color: '#4D4D4C',
+		},
+	],
+} as const;
+
+/**
+ * Based on Dracula theme by Zeno Rocha
+ * https://github.com/vadimdemedes/thememirror/blob/main/source/themes/dracula.ts
+ */
+const DARK_THEME = {
```
Comment (mutdmour) [inline packages/editor-ui/src/components/CodeNodeEditor/theme.ts:81]: can you try implementing this using css variables instead?

**Review by mutdmour — COMMENTED** [review]
thanks.. love the updates to the code node.. I have setup a call with design to see how we can introduce Dark mode completely as a Beta feature...

**Review by mutdmour — APPROVED** [review]

**Comment by janober** [comment]
Got released with `n8n@0.200.0`

**Pull request merged**